src/Symfony/Component/HttpClient/Response/JsonMockResponse.php

```diff
@@ -15,6 +15,9 @@
 
 class JsonMockResponse extends MockResponse
 {
+    /**
+     * @param mixed $body The json response body to encode
```

Comment:
It looks like redundant PHPDoc. Do you mean the real arg type does not have priority over the inherited PHPDoc?
Which SCA tool reports that plz?